### PR TITLE
Make script parameterless

### PR DESCRIPTION
**Pull request by gkanade**

### CODE REVIEW

File: zeppelin/hdi_add_zeppelin.sh [hunk 1]
```diff
@@ -3,127 +3,57 @@
 #----------------------------------------------------------------
 # THIS SCRIPT ADDS INSTALLS AND STARTS ZEPPELIN ON HBASE CLUSTER 
 #----------------------------------------------------------------
+#Import helper module
+wget -O /tmp/HDInsightUtilities-v01.sh -q https://hdiconfigactions.blob.core.windows.net/linuxconfigactionmodulev01/HDInsightUtilities-v01.sh && source /tmp/HDInsightUtilities-v01.sh && rm -f /tmp/HDInsightUtilities-v01.sh
```
Comment (dulems) [inline zeppelin/hdi_add_zeppelin.sh:7]: are you using any of these utilities? It's better to not take dependencies on this if you don't need it.

File: zeppelin/hdi_add_zeppelin.sh [hunk 2]
```diff
@@ -3,127 +3,57 @@
...
-		;;
-            *)
-                break
-        esac
-        shift
-    done
+checkHostNameAndSetClusterName() {
+    HEADNODE_HOST_FQDN=$(hostname -f)
+    echo "HEADNODE_HOST_FQDN=$HEADNODE_HOST_FQDN. Lower case: ${HEADNODE_HOST_FQDN,,}"
+    if ! [[ $HEADNODE_HOST_FQDN == hn0* ]]; then
```
Comment (dulems) [inline zeppelin/hdi_add_zeppelin.sh]: because of overprovisioning, headnode0 might not even exist. What version of the Hue script were you referring to?

**Review by dulems — COMMENTED** [review]
comment

File: zeppelin/hdi_add_zeppelin.sh (hunk 1 above)
Comment (gkanade) [reply zeppelin/hdi_add_zeppelin.sh:7]: sorry forgot to remove this; initially i was planning on using this but not anymore

File: zeppelin/hdi_add_zeppelin.sh (hunk 2 above)
Comment (gkanade) [reply zeppelin/hdi_add_zeppelin.sh]: we do not overprovision headnode afaik

File: zeppelin/hdi_add_zeppelin.sh (hunk 1 above)
Comment (gkanade) [reply zeppelin/hdi_add_zeppelin.sh:7]: Readded this now that we are using the utilities

File: zeppelin/hdi_add_zeppelin.sh (hunk 2 above)
Comment (gkanade) [reply zeppelin/hdi_add_zeppelin.sh]: added code to guard against possible overprovisoning

**Review by dulems — APPROVED** [review]

**Comment by gkanade** [comment]
Thanks for the reviews. This is now made parameterless.

**Pull request merged**